openedx/features/classrooms/apps.py

```diff
@@ -1,8 +1,14 @@
+"""
+Apps file for classrooms application contains configuration
+"""
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
 from django.apps import AppConfig
 
 
 class ClassroomsConfig(AppConfig):
+    """
+    A default configurations for classrooms application
```

Comment:
Let's change it to: "Default configuration for classrooms application"

Reply:
Done